wiki/es/es-Installation-for-Linux.md

```diff
@@ -29,22 +29,16 @@ tar -xvf latest.tar.gz
 
 Primero actualiza la lista de paquetes (por ej. en distribuciones basadas en Debian con `sudo apt-get update`).
 
-En distros basadas en **Ubuntu**  18.04+ y en Debian 9 o 10:
+En distros basadas en **Ubuntu**  18.04+, **Debian** 9+ o 10 y Raspberry Pi Raspbian Buster o superior:
 
 ```shell
-sudo apt-get install build-essential qt5-qmake qtdeclarative5-dev qt5-default qttools5-dev-tools libqt5concurrent5 libjack-jackd2-dev
-```
-
-En **Debian** 11 (bullseye) o posteriores, y la versión de Raspberry Pi Raspbian Buster o posterior:
-
-```shell
-sudo apt-get install build-essential qtdeclarative5-dev qt5-default qttools5-dev-tools libqt5concurrent5 libjack-jackd2-dev
+sudo apt-get install build-essential qt5-qmake qtdeclarative5-dev qt5-default qttools5-dev-tools libjack-jackd2-dev 
 ```
 
 En **Fedora**:
 
 ```shell
-sudo dnf install qt5-qtdeclarative-devel jack-audio-connection-kit-dbus libqt5concurrent5 jack-audio-connection-kit-devel
```

Comment:
why has this been removed?

Reply:
The dependency list? If you mean that, it's in the document - I don't know why it's recorded as having been erased, but it's there (I just checked).

Reply:
Ok. So I will review it and if it's removed will re-add it.